cla_public/apps/checker/fields.py

```diff
@@ -161,10 +161,22 @@ def __init__(self, label=None, validators=None, min_val=0, max_val=9999999999, *
         self.min_val = min_val
         self.max_val = max_val
 
+    def convert_amount_to_utf8(self, amount):
+        # Covert pounds side of the input into UTF-8 format
+        try:
+            amount = amount.decode("utf-8")
+        except UnicodeEncodeError:
+            # Input is already in UTF-8 format
```

Comment:
Is this comment the right way around?

Reply:
I believe so as this is why the decoding fails

Reply:
I thought `.decode("utf-8")` meant decode *from* utf-8, not *to* utf-8, in which case I would expect it failing to mean it's no longer in utf-8

Reply:
Ironically after doing some research `.decode("utf-8")` means to turn a string of bytes to a unicode string. It works the opposite way around to what you would except..

So when it meets a unicode string and it tries to decode it, it is already in unicode and not bytes, so it wouldn't work. Also learnt Python tries to convert (encode) the string to ASCII to help us so that we can decode it, but £ is a non-ASCII character resulting in the `UnicodeEncodeError`